### DISCUSSION TITLE
New vehicle for Webots?

### DISCUSSION THREAD

**Discussion by harunkurtdev** [cyberbotics/webots#6718] [Ideas]
Mr. @lukicdarkoo,

Does it make sense to create Blue Robotics BlueBoat for the Webots ?

USV details belong to in this [link](https://bluerobotics.com/store/blueboat/blueboat/)

![image](https://github.com/user-attachments/assets/0d6f1d7a-d98a-4a41-b705-6d35b2b10289)

![image](https://github.com/user-attachments/assets/69449d2b-dee6-4207-bd17-84fe765bcc6c)

**Comment by harunkurtdev** [comment]
I built

https://github.com/user-attachments/assets/1b0f39ac-03c4-4650-8a21-8007a4a4e474

**Comment by lukicdarkoo** [comment]
Very cool. I believe it is the best to create a separate repo and we can feature it.
